release.auto.pkrvars.hcl

```diff
@@ -1,8 +1,8 @@
-ami_version        = "20220121"
-source_ami_al2     = "amzn2-ami-minimal-hvm-2.0.20211223.0-x86_64-ebs"
-source_ami_al2arm  = "amzn2-ami-minimal-hvm-2.0.20211223.0-arm64-ebs"
-ecs_agent_version  = "1.58.0"
-ecs_init_rev       = "2"
+ami_version        = "20220209"
+source_ami_al2     = "amzn2-ami-minimal-hvm-2.0.20220207.1-x86_64-ebs"
+source_ami_al2arm  = "amzn2-ami-minimal-hvm-2.0.20220207.1-arm64-ebs"
+ecs_agent_version  = "v1.59.0"
```

Comment:
do we need the leading 'v' here?

Reply:
previous rev didn't have the "v"

Reply:
I just ran the script as it was written out in the MCM, I'll take it out though!